SemontoFramework/src/ServerHealth/functions/functions.php

```diff
@@ -31,7 +31,7 @@ function getTests ($config, $db) {
                 $include_path = __DIR__ . $include_path_default_tests . $test_class . '.php';
             } else {
                 header('HTTP/1.1 500 Internal Server Error');
-                exit("Test not found ($test_class)");
```

Comment:
Mag een continue hier, we geven wel een HTTP 500 maar de JSON die mogelijks al opgebouwd is, gaat andere testen geven. Confusing. Moeten we beter opvangen denk ik? Mss moeten we hier bv eerder Exceptie opgooien? Die we hieronder opvangen, en waar je al return [] doet?